packages/react-tinacms-editor/src/components/BaseMenubar/index.tsx

```diff
@@ -37,6 +43,24 @@ interface Props {
   popups?: ReactElement[]
 }
 
```

Comment:
Function to calculate offsetTop is already in utils: https://github.com/tinacms/tinacms/blob/master/packages/react-tinacms-editor/src/utils.ts#L21